### PR TITLE
SAMBRO: edit approval message + SAMBRO: fix translation problem + SAMBRO: tweaks

### PR DESCRIPTION
**Pull request by biplovbhandari**
@flavour Please merge

### CODE REVIEW

File: modules/s3db/cap.py [hunk 1]
```diff
@@ -59,7 +59,6 @@
 from gluon.tools import fetch
 from ..s3 import *
 
-T = current.T
```
Comment (flavour) [inline modules/s3db/cap.py]: Good, but you'll need to do a lot more, no?
All the cap_incident_type_opts justunder this refer to T which will no longer be available...these will need moving into a class...can be a small class with just these to minimise need for loading table models unnecessarily.

File: controllers/cap.py [hunk 2]
```diff
@@ -1287,7 +1299,16 @@ def notify_approver():
                 subject = "%s: Alert Approval Required" % settings.get_system_name_short()
                 url = "%s%s" % (settings.get_base_public_url(),
                                 URL(c="cap", f="alert", args=[alert_id, "review"]))
-                message = "You are requested to take action on this alert:\n\n%s" % url
+                message = """
+Hello Editor,
+%(first_name)s %(last_name)s has created the alert message.
```
Comment (flavour) [inline controllers/cap.py]: Much nicer...although I would suggest using s3_fullname if possible...this supports Middle Names too (& we can adjust word order based on locale in future)

File: modules/s3db/cap.py (hunk 1 above)
Comment (nursix) [inline modules/s3db/cap.py]: Or, you simply remove the T's in these option dicts, and instead add translate=True to the respective S3Represent instances.

Comment (flavour) [inline modules/s3db/cap.py]: Also used in S3OptionsFilter options

Comment (nursix) [inline modules/s3db/cap.py]: Ah...doesn't really hurt to add a translate-option to the S3OptionsFilter with fixed options dicts as well, does it? Maybe I should do that...

**Pull request closed without merging**